_posts/2022-03-25-two-player-auction-learning.md

```diff
@@ -0,0 +1,301 @@
+---
+ layout: post
+ title: '"Auction Learning as a Two Player Game": GANs (?) for Mechanism Design'
+ tags: [Mechanism Design, Deep Learning, Auctions]
+ authors: Curry, Michael J., University of Maryland; Reusche, Daniel
+---
+
```

Comment:
Could you add a short summary/abstract before the `# Introduction` header? As it stands, your post renders like the image below, and I think it would be better to have a brief summary

![image](https://user-images.githubusercontent.com/19653980/163032475-c82726b3-cb73-40c4-9394-71dca66a9a71.png)